hugr-py/tests/test_hugr_build.py

```diff
@@ -404,3 +405,47 @@ def test_option() -> None:
     dfg.set_outputs(b)
 
     validate(dfg.hugr)
+
+
+# a helper for the toposort tests
+def _build_test_hugrfn() -> Function:
```

Comment:
You can mark this as a fixture,
```suggestion
@pytest.fixture
def simple_function() -> Function:
    """A helper for the toposort tests."""
```

And then use it as a parameter to the tests
```python
def test_toposort(simple_function: Function) -> None: ...
```